README.md

```diff
@@ -66,19 +66,18 @@ To learn more about the different rule types, their guarantees and how they are
 #### Carrying out software supply chain steps
 
 ##### in-toto-run
-`in-toto-run` generates link metadata for the given command-line option and runs it as its own command. See the [this simple usage example from the demo application for more details](https://github.com/in-toto/demo#tampering-with-the-software-supply-chain).
 
-```shell
-in-toto-run  --step-name <unique step name>
-            {--key <functionary signing key path>,  --gpg [<functionary gpg signing key id>]}
-            [--gpg-home <path to gpg keyring>]
-            [--materials <filepath>[ <filepath> ...]]
-            [--products <filepath>[ <filepath> ...]]
-            [--record-streams]
-            [--no-command]
-            [--verbose] -- <cmd> [args]
-```
+`in-toto-run` is used to execute one of the software supply chain steps. The 
+relevant step name and command are passed as arguments, along with materials, 
+which are files required for command execution, and products which are files
+expected as a result of the execution of the command. These, and other 
+relevant details pertaining to the step are stored in a link file, which is 
+signed using the functionary's key.
 
+See [this simple usage example from the demo application 
+for more details](https://github.com/in-toto/demo). 
+For a detailed list of all the command line arguments, run `in-toto-run --help` 
+or look at the code documentation [here](https://github.com/in-toto/in-toto/blob/develop/in_toto/in_toto_run.py).
```

Comment:
Please consider the comment I left about link tests (here and below).

Reply:
Do you mean the web layout wizard specified below?

Reply:
Oh do you mean this: 

> It's generally good (accessibility) practice to have link texts describe what they are linking to, e.g.
"...code documentation..." instead of "..documentation here...".

Reply:
Fixed. I was confused by tests.